src/utils/test-utils.js

```diff
@@ -9,3 +10,13 @@ export async function renderAndWait (ui, ms = 0, options = {}) {
   })
   return queries
 }
+
+export async function renderAndWaitWithModal (ui, ms = 0, options = {}) {
```

Comment:
I feel ambivalent about this. I get that it's saving loc, but it's not really a logical unit and I think I would prefer to keep the two parts separate.

Reply:
I can move it into each of the test files, but I'd also be happy including both in utils, but distinguishing between helper functions and functions shared across multiple files.  Thoughts?

Reply:
Let's talk about this together and come to a good rule thumb. :+1: :eyeglasses: